### PR TITLE
Import alertmanager docs

### PR DESCRIPTION
**Pull request by josedonizetti**
This is part of the ongoing work to versioned prometheus documentation.
https://github.com/prometheus/docs/issues/857

@grobie

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+# Alertmanager 
+
+Welcome to Alertmanager documentation.
+
+## Content
+
+- [Overview](overview.md)
+- [Introduction](introduction.md)
+- [Configuration](configuration.md)
+- [Clients](clients.md)
```
Comment (brian-brazil) [inline docs/index.md]: Clients should go last, that's a technical reference very very few people need.

File: docs/index.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+- [Notifications](notifications.md)
```
Comment (brian-brazil) [inline docs/index.md]: The word Template should be in here

File: docs/index.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
+---
```
Comment (brian-brazil) [inline docs/index.md]: Are the indexes not automatically generated?

Comment (josedonizetti) [reply docs/index.md]: @brian-brazil not sure about it. I didn't get the idea that the index.md on the promo repo was generated. Is there any reference about this?

Comment (brian-brazil) [reply docs/index.md]: If you look at them they're just the section title, with nothing else.

Comment (josedonizetti) [reply docs/index.md]: @grobie how did you generate the index.md on the prometheus doc?

Comment (grobie) [reply docs/index.md]: I wrote the `index.md` by hand actually. My intent was to give users who read the docs locally a starting point. It was not meant to be just a list of the pages, but to include a small introduction as well. We can change that, but I guess a little explanation that the main documentation lives on a page online would be nice.

Comment (josedonizetti) [reply docs/index.md]: @brian-brazil @grobie should I add a brief content to it, or just remove it for now?

Comment (grobie) [reply docs/index.md]: I'd keep it as starting point. But I'm fine with either.

File: docs/clients.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
+---
+title: Clients
+sort_rank: 6
+nav_icon: sliders
```
Comment (grobie) [inline docs/clients.md]: Remove all the `nav_icon` settings please.

**Comment by josedonizetti** [comment]
@grobie Done! Question, after this is merged how will be the process to update the documentation?
When a PR is merged to change the documentation here, will the docs project rebuild itself or something else is needed?

**Comment by grobie** [comment]
We use https://www.netlify.com/ now to build and host the docs. They provide webhooks and I configured prometheus/prometheus to use the trigger that on every push. I'll do the same for alertmanager.

I need to make some changes to the docs repo to support multiple versioned repo docs. Took some shortcuts on the way.

**Comment by josedonizetti** [comment]
@grobie 

@stuartnelson3 is cutting a new release for alertmanger, and there are some important changes that must be updated in the documentation (pagerduty, and opsgenie were upgraded to a new version of their APIs) https://github.com/prometheus/alertmanager/pull/1089

Do we change the existing docs so users have it properly there, and later update it here, or the change you will do on the docs repo to support alertmanager is coming soon and we wait to update it only here?

**Comment by grobie** [comment]
I'm almost done with the nanoc4 upgrade. I hope to have the changes ready tonight or latest Saturday night.

**Comment by stuartnelson3** [comment]
@grobie ping

**Comment by grobie** [comment]
ack

**Comment by josedonizetti** [comment]
@grobie Is this doc migration for alertmanager still happening?

**Comment by aioue** [comment]
any way I can help progress this merge?

**Comment by grobie** [comment]
Sorry for the delays. I've been busy with other projects recently. The
blocker is a upgrade and rewrite of the code in prometheus/docs. I'm 70%
done with that, but haven't had a chance to look into that again. I'll try
to reserve some time over the weekend and get that done, so that you can
update the docs here.

**Comment by stuartnelson3** [comment]
@grobie status on this?

**Comment by juliusv** [comment]
@grobie Ping. Status?

**Comment by grobie** [comment]
No progress. Depends on a change and rewrite of the docs infrastructure. I'll try to pick that up again during the next two weeks, but can't guarantee anything.

**Comment by juliusv** [comment]
@grobie Is this something that Luc (our new docs person) could help with, or are you too far into it?

**Comment by roidelapluie** [comment]
Superseeded by #2292 , and https://github.com/prometheus/docs/pull/1657

**Comment by josedonizetti** [comment]
Considering @roidelapluie work, I’m closing this PR.

**Pull request closed without merging**